static/engine/core/index.js

```diff
@@ -15,10 +15,12 @@ ErrorHandler.run(() => {
 	const allGameObjects = new AllGameObjects()
 
 	const levelSelector = new LevelSelector()
-	levelSelector.changeActiveLevel(new World(levelSelector, allGameObjects, camera, mouse, controller))
-	// levelSelector.changeActiveLevel(new WorldEditor(camera, mouse))
+	//levelSelector.changeActiveLevel(new World(levelSelector, allGameObjects, camera, mouse, controller))
+	levelSelector.changeActiveLevel(new WorldEditor(camera, mouse))
 
-	// Overlay.create()
+	Overlay.create(onload =  () => {
+		ImageSelectorProvider.addImageSelector()
```

Comment:
don't do it this way. this is just unnecessarily complicated

Reply:
i did it this way , coz the image selector element has to be loaded , only after html is loaded in overlay.create() 
so i added a callback hook parameter that will be called after the inner html loaded . 
and we also need the selected element information in world editor . 
this way it is more loosely coupled from either of the class.

Reply:
do it the simpler way. no need to overcomplicate things

Reply:
done